webpack/webpack.dev.js

```diff
@@ -12,23 +12,22 @@ module.exports = {
   },
   module: {
     rules: [{
-        test: /\.(css|scss)$/,
-        use: [
-          'style-loader',
-          {
-            loader: 'css-loader',
-            options: { 
-              sourceMap: true,
-              localsConvention: 'camelCase',
-              modules: {
-                localIdentName: '[local]___[hash:base64:5]'
-              }
+      test: /\.(css|scss)$/,
+      use: [
+        'style-loader',
+        {
+          loader: 'css-loader',
+          options: {
+            sourceMap: true,
+            localsConvention: 'camelCase',
+            modules: {
+              localIdentName: '[local]___[hash:base64:5]'
             }
-          },
-          'sass-loader'
-        ]
-      }
-    ]
+          }
+        },
+        'sass-loader'
+      ]
+    }]
   },
   devServer: {
     historyApiFallback: true,
```

Comment:
Something strange here. When I reload the page on http://baseurl/world/:cityId , i'm receiving 404. But if url is only http://baseurl/world app reloading correctly.